src/Reflection/ReflectionReturnValue.php

```diff
@@ -16,78 +18,38 @@
 class ReflectionReturnValue
 {
     /**
-     * Return value type
      * @var string
      */
     protected $type;
 
     /**
-     * Return value description
      * @var string
      */
     protected $description;
 
-    /**
-     * Constructor
-     *
-     * @param string $type Return value type
-     * @param string $description Return value type
-     */
-    public function __construct($type = 'mixed', $description = '')
+    public function __construct(string $type = 'mixed', string $description = '')
     {
         $this->setType($type);
         $this->setDescription($description);
     }
 
-    /**
-     * Retrieve parameter type
-     *
-     * @return string
-     */
-    public function getType()
+    public function getType(): string
     {
         return $this->type;
     }
 
-    /**
-     * Set parameter type
-     *
-     * @param string|null $type
-     * @throws Exception\InvalidArgumentException
-     * @return void
-     */
-    public function setType($type)
+    public function setType(?string $type = null): void
```

Comment:
the same comments as in the previous class here

Reply:
Changed.